src/Grid/utils.jl

```diff
@@ -340,3 +340,28 @@ for (func,                   entity_f,            entity_t,     filter_f,
     end
 end
 end
+
+# Hypercube is simply ⨂ᵈⁱᵐ Line :)
+sample_random_point(::Type{Ferrite.RefHypercube{ref_dim}}) where {ref_dim} = Vec{ref_dim}(2.0 .* rand(Vec{ref_dim}) .- 1.0)
```

Comment:
To avoid the array allocation
```suggestion
sample_random_point(::Type{Ferrite.RefHypercube{ref_dim}}) where {ref_dim} = Vec{ref_dim}(i -> 2 * rand() - 1)
```